src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -108,7 +110,8 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
                     final Entity entityCitizen = world.getEntityByID(citizenDataView.getEntityId());
                     if (entityCitizen instanceof EntityCitizen && entityCitizen.getPosition().distanceSq(player.getPosition()) <= 2)
                     {
-                        RenderUtils.renderSigns(world, event.getPartialTicks(), citizenDataView, player, entityCitizen.getPosition());
+                        //TODO: Reimplement sign rendering.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)